CHANGELOG.md

```diff
@@ -9,41 +9,89 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 Note: Changes are now automatically tracked in [GitHub](https://github.com/sqlfluff/sqlfluff/releases) and will be copied in here on each release (please remember to update the issues and contributors to links!). There is no need to manually edit this file going forward.
 -->
 
+## [0.6.7] - 2021-10-04
+
+Lots of bug fixes to our rules (particularly the autofixer), plus good improvements to Exasol, Snowflake, and T-SQL, dialects amongst others! Plus we added Hive and SQLite as supported dialects!
```

Comment:
Re: The autofixer, maybe mention that many of the fixes address improve handling of Jinja and dbt templating.

Reply:
Good idea. Changed it to:

> Lots of fixes to our rules (particularly when running `sqlfluff fix`, and particularly for Jinja and DBT templates). We also have good improvements to Exasol, Snowflake, and T-SQL dialects amongst others. Plus we added Hive and SQLite as supported dialects!